src/treq/test/local_httpbin/test/test_child.py

```diff
@@ -263,14 +261,13 @@ def flush(self):
         self._state.flush_count += 1
 
 
-if not six.PY2:
-    @attr.s
-    class BufferedStandardOut(object):
-        """
-        A standard out that whose ``buffer`` is a
-        :py:class:`FlushableBytesIO` instance.
-        """
-        buffer = attr.ib()
+@attr.s
+class BufferedStandardOut(object):
```

Comment:
Can kill the `object` here and in other places now too.

Reply:
Done.